cooltools/api/expected.py

```diff
@@ -1017,25 +1019,43 @@ def expected_cis(
 
     # additional smoothing and aggregating options would add columns only, not replace them
     if smooth:
+        if clr_weight_name is None:
+            # result["count.avg"] = result["count.sum"] / result["n_valid"]
+            cols = {
+                "dist": "dist",
+                "n_pixels": _NUM_VALID,
+                "n_contacts": "count.sum",
+                "contact_freq": "count.avg",
+                "smooth_suffix": ".smoothed",
```

Comment:
Consider renaming "suffix", "smooth suffix seems rather confusing and hard to generalize in future.